plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -42,6 +44,29 @@ export class TeamManager {
         }
     }
 
+    async flushLastSeenAtCache(): Promise<void> {
+        const team_ids: string[] = []
+        const event_names: string[] = []
+        const last_seen_at_array: number[] = []
+
+        for (const event of this.eventLastSeenCache) {
+            team_ids.push(event[0].split('_')[0])
+            event_names.push(event[0].split('_')[1])
```

Comment:
This doesn't work - event names _will_ have any character in it, so this will have bugs.

Reply:
I've already changed this to just grab the first `_`